### ISSUE TITLE
List of non-conforming FMUs regarding 'overstepping' end time 

### ISSUE THREAD

**Issue by ghorwin** [modelica/fmi-cross-check#64]
According to recent discussion/clarification on standard interpretation, CS-FMUs must be able to handle "doStep()" calls gracefully, even if  `t+h > tEnd`. This may happen due to rounding errors for both fixed and variable step FMUs. See also discussion in ticket https://github.com/modelica/fmi-standard/issues/575

Currently, there are a few FMUs in the cross-check repo, that fail the last step when end time is exceeded. Importing tools cannot comply with those test cases, unless they ignore the "constant step size" property and adjust the last step manually. Therefore it may be meaningful to update these FMUs and in the mean time mark them 'notCompliantWithLatestRules'.

Here's a list of tools that export FMUs with strict end-time checking:
* 20Sim
* MapleSim

(not complete, yet)

**Comment by lochel** [comment]
@ghorwin can you please open a pull request adding the `notCompliantWithLatestRules` to the examples you identified?

**Comment by ghorwin** [comment]
I may need a few days to look into it. The file should just be "there", besides the `fmu`-file, right? No special content is required?

**Comment by lochel** [comment]
Yes, exactly. It is just an empty file as of now. Thanks for your contribution.
